projects/developer/src/app/common/components/live-range-selector/component.scss

```diff
@@ -0,0 +1,77 @@
+.live-range-selector {
+    align-items: flex-end;
+
+    div {
+        margin-right: 20px;
+    }
+
+    .ui-inputgroup {
+        .ui-inputgroup-addon {
+            line-height:  normal;
+        }
+
+        ::ng-deep input,
+        ::ng-deep .ui-dropdown {
+            line-height:  normal;
+            border-top-left-radius: 0;
+            border-bottom-left-radius: 0;
+            flex: 1 1 auto;
+        }
+    }
+
+    .live-range-selector__date-filter-btn, .live-range-selector__date-filter-dropdown {
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+
+    &.spaced {
+        .live-range-selector__date-filter-ranges, .live-range-selector__date-filter-dropdown {
+            margin-left: 55px;
+
+            button {
+                margin-left: 3px;
+            }
+        }
+    }
+
+    .live-range-icon {
+        margin-left: 5px;
+        margin-right: 5px;
+    }
+
+    .live-range-inactive {
+        color:grey
+    }
+
+    .live-range-active {
+        animation:greenBlinking 1.5s infinite;
+    }
+
+    .live-range-error {
+        animation:redBlinking 1.5s infinite;
+    }
+}
+
+@media screen and (max-width: 858px) {
+    .live-range-selector__date-filter-btn, .live-range-selector__date-filter-dropdown, .live-range-selector__date-filter-ranges {
+        margin-left: 0 !important;
+        padding: 0 !important;
+    }
+
+    .live-range-selector__date-filter-dropdown, .live-range-selector__date-filter-btn {
+        margin-top: 5px !important;
+        margin-right: 5px !important;
+    }
+}
+
+@keyframes redBlinking{
+    0%{     color: #777;    }
+    50%{    color: var(--red); }
+    100%{   color: #777;    }
+}
+
+@keyframes greenBlinking{
```

Comment:
Curious about preference of 0 -> 50% ->100% vs `from -> to`
```scss
.live-range-active {
    animation: greenBlinking 0.75s alternate ease-in-out infinite;
}

@keyframes greenBlinking{
    from {    color: #777;    }
    to   {    color: var(--green); }
}
```

Reply:
Despite this looking like a new component, this was split out from the temporal filter component. I'm not sure who wrote this.